README.md

```diff
@@ -1,33 +1,18 @@
 ## Codecademy React Challenge Project
 
-This project will ask the learner to design an appointment app that resembles those of medical offices like LabCorp and Quest Diagnostics. The app will allow users logged in as Providers to add appointment availability and users logged in as Patients" to reserve an available appointment. 
+#### Notes 2/16/21
+- react-datepicker: I was able to grab the css from *node_modules* for this component and put it in **index.html**. Will the LE have access to it the react-datepicker component though? I will create a date validation field if necessary.
```

Comment:
Using this seems like it'll introduce too many headaches down the road. Can we use `<input type="date" />` instead? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date

Reply:
Much easier. Done.